contrib/opencensus-ext-stackdriver/tests/test_stackdriver_exporter.py

```diff
@@ -678,12 +678,12 @@ def test_monitored_resource_attributes_gke(self, gmr_mock):
         }
 
         mock_resource = mock.Mock()
-        mock_resource.get_type.return_value = 'gke_container'
+        mock_resource.get_type.return_value = 'k8s_container'
         mock_resource.get_labels.return_value = {
-            'pod_id': 'pod',
-            'cluster_name': 'cluster',
-            'namespace_id': 'namespace',
-            'container_name': 'c1',
+            'k8s.io/pod/name': 'pod',
+            'k8s.io/cluster/name': 'cluster',
+            'k8s.io/namespace/name': 'namespace',
+            'k8s.io/container/name': 'c1',
             'project_id': 'my_project',
             'instance_id': 'instance',
```

Comment:
Nit: `instance_id` shouldn't be part of `k8s_container`

Reply:
Is instance_id not part of the Host?

Reply:
It is, but in Stackdriver definition `k8s_container` resource doesn't have this label. Setting `instance_id` for `k8s_container` will result in errors (e.g https://github.com/census-instrumentation/opencensus-java/issues/1888).

Reply:
ACK